File: Aula4/ComputationalThinkingAula4/src/exercicio4.java
import java.util.Scanner;

public class exercicio4 {

	public static void main(String[] args) {
		//Entrada: O usuário digitará a cotação do dólar (1$ = xR$) e o seu montante em reais (mR$).
		//Processamento: Realizar a conversão do montante de reais do usuário em dólar (1$*mR$/xR$ = q$).
		//Saída: Exibir o valor em dólar (q$) da quantia em reais que o usuário têm.
		
		
		float dolar, montante, quantia;
		
		Scanner leitor = new Scanner (System.in);
		
			System.out.println("Digite a cotação do dolar hoje ");
			dolar = leitor.nextFloat();
			System.out.println("Digite o seu montante em reais ");
			montante = leitor.nextFloat();
			
			leitor.close();
			
			quantia = montante/dolar;
		
			System.out.println("Valor do seu montante em dolar é " +quantia);
	}

}

File: aula5e6/exerciciosPython/exercicio4.py
''' O usuário digita quanto está valendo o
dólar e quanto em reais ele possui. O programa exibe quantos dólares
vale os reais que o usuário informou'''

dolar = int(input("Digite o valor da cotação do dólar hoje: "))
montante = int(input("Digite o valor em reais do seu montante: "))

quantia = montante/dolar

print("A sua quantia equivale a {}".format(quantia)) #impressão do resultado
print("A sua quantia equivale a", quantia)